### PR TITLE
Add post mortem and date previous post mortem

### PR DESCRIPTION
**Pull request by kelsonic**
This PR adds a post mortem for the Cerner issue that occurred yesterday.

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]

File: products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- [Off-schedule deploy](https://dsva.slack.com/archives/CBU0KDSB1/p1615233939312300)
+
+## Action Items
+
+| Description                    | Type    | Owner        | Issue # |
+| ------------------------------ | ------- | ------------ | ------- |
+| We should add a unit test [here](https://github.com/department-of-veterans-affairs/vets-website/blob/dbd95d9098ccd5acc1787ceb9ee3fc55da3e7852/src/platform/utilities/cerner/index.js#L5) to ensure that any Cerner facility ID that is added to `vets-website` has a related feature toggle [here](https://github.com/department-of-veterans-affairs/vets-website/blob/3f41da4363074b54dc3efab7baff0f4898bb47a4/src/platform/utilities/feature-toggles/featureFlagNames.js#L67) | Frontend | Public Websites | Ticket not created yet |
+| We should have __at least__ 1 test account on a non-production environment that belongs to all Cerner facilities. | Database | Unknown | Ticket not created yet |
+| We should create a step-by-step guide to adding support for a facility to become a Cerner facility that documents all necessary steps that have to be taken. | Frontend | Public Websites | Ticket not created yet |
```
Comment (kelsonic) [inline products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md:111]: @brianalloyd we may need a ticket for this

File: products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+- [`vets-website` PR to add Cerner facility IDs (culprit)](https://github.com/department-of-veterans-affairs/vets-website/pull/16220)
+- [Detection](https://dsva.slack.com/archives/C52CL1PKQ/p1615231277236500)
+- [`vets-website` PR to add missing feature toggles (fix)](https://github.com/department-of-veterans-affairs/vets-website/pull/16295)
+- [Off-schedule deploy](https://dsva.slack.com/archives/CBU0KDSB1/p1615233939312300)
+
+## Action Items
+
+| Description                    | Type    | Owner        | Issue # |
+| ------------------------------ | ------- | ------------ | ------- |
+| We should add a unit test [here](https://github.com/department-of-veterans-affairs/vets-website/blob/dbd95d9098ccd5acc1787ceb9ee3fc55da3e7852/src/platform/utilities/cerner/index.js#L5) to ensure that any Cerner facility ID that is added to `vets-website` has a related feature toggle [here](https://github.com/department-of-veterans-affairs/vets-website/blob/3f41da4363074b54dc3efab7baff0f4898bb47a4/src/platform/utilities/feature-toggles/featureFlagNames.js#L67) | Frontend | Public Websites | Ticket not created yet |
```
Comment (kelsonic) [inline products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md:109]: @brianalloyd we may need a ticket for this

Comment (brianalloyd) [reply products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md:109]: @kelsonic Ticket #21235 to address the above.

File: products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md (hunk 1 above)
Comment (brianalloyd) [reply products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md:111]: @kelsonic ticket #21238 to address top issue.  And ticket #21239 to address the second issue.

**Pull request merged**